presenter/src/components/sections/FourthSection.test.ts

```diff
@@ -1,12 +1,56 @@
-import { mount } from '@vue/test-utils'
-import { describe, it, expect } from 'vitest'
+import { VueWrapper, mount } from '@vue/test-utils'
+import { describe, it, expect, beforeEach } from 'vitest'
+import { ComponentPublicInstance } from 'vue'
 
 import FourthSection from './FourthSection.vue'
 
-describe('renders FourthSection', () => {
-  const wrapper = mount(FourthSection)
+describe('FourthSection', () => {
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+  let wrapper: VueWrapper<unknown, ComponentPublicInstance<unknown, Omit<unknown, never>>>
+  const Wrapper = () => {
+    return mount(FourthSection)
+  }
```

Comment:
```suggestion
  const Wrapper = () => {
    return mount(FourthSection)
  }
  let wrapper: ReturnType<typeof Wrapper>
```

See other examples - search for `ReturnType<typeof` to find examples if this is not working out of the box